__tests__/app.test.js

```diff
@@ -210,7 +210,15 @@ describe("POST", () => {
   });
 });
 
-
+describe("DELETE", () => {
+    describe('/api/comments/:comment_id', () => {
+        test('Status: 204', () => {
+            return request(app)
+            .delete("/api/comments/2")
+            .expect(204)
```

Comment:
Nice! Could also check the database in the tests to see if the comment has actually been deleted! Up to you though!